apps/api/v2/src/modules/organizations/controllers/outputs/listUsersResponse.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { IsArray } from "class-validator";
+
+export interface User {
+  id: number;
+  name: string | null;
+  email: string;
+  timeZone: string;
+}
+
+export class ListUsersResponseDto {
+  @IsArray()
```

Comment:
When using "@IsArray" with an object type like "User", the setup should look like:
```
@ValidateNested()
@Type(() => User)
@IsArray()
users!: User[];
```